docs/examples/composer_app/src/app.php

```diff
@@ -0,0 +1,26 @@
+<?php
```

Comment:
Why not just move these two files (app.php and log4php.properties) into the parent directory?